app/controllers/providers/full_employment_details_controller.rb

```diff
@@ -18,5 +21,21 @@ def form_params
         params.require(:legal_aid_application).permit(:full_employment_details)
       end
     end
+
+    def message_sentry
+      Sentry.capture_message("HMRC response still pending: correlation id: #{correlation_id}")
```

Comment:
it looks you will never get a correlation id from a pending/processing response so you might want to remove the correlation_id and related code, here or after the fact